src/common/sanity-utils.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * (c) 2021, Micro:bit Educational Foundation and contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+import { ToolkitPortableText } from "../documentation/explore/model";
+
+const defaults = { nonTextBehavior: "remove" };
+
+export const blocksToText = (
```

Comment:
Let's test this.

Reply:
[`e1a4bf1`](https://github.com/microbit-foundation/python-editor-next/pull/452/commits/e1a4bf1ce1d6abe301056f5fa8581e7123354a49)